File: src/main/java/com/jobportel/boot/model/Result.java
package com.jobportel.boot.model;

import lombok.Data;

@Data
public class Result {
    private Long id;
    private String candidateEmail;
    private int aptitudeCorrect;
    private int reasoningCorrect;
    private int communicationCorrect;
    private int totalCorrect;
    private double percentage;
}

File: src/models/Result.js
export class Result {
  constructor(data = {}) {
    this.id = data.id || null;
    this.candidateEmail = data.candidateEmail || '';
    this.aptitudeCorrect = data.aptitudeCorrect || 0;
    this.reasoningCorrect = data.reasoningCorrect || 0;
    this.communicationCorrect = data.communicationCorrect || 0;
    this.totalCorrect = data.totalCorrect || 0;
    this.percentage = data.percentage || 0;
  }
}
